client/src/pages/Settings/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -0,0 +1,98 @@
+import { Typography } from '@mui/material';
+import { Box } from '@mui/system';
+import { Button } from '@mui/material';
+import { Input } from '@mui/material';
+import { Avatar } from '@mui/material';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTrashAlt } from '@fortawesome/free-solid-svg-icons';
+
+interface ProfilePhotoProps {
+  header: string;
+  imageUrl: string;
+}
+
+const openFileSelector = () => {
+  document.getElementById('photoInput')?.click();
+};
+
+const ProfilePhoto: React.FC<ProfilePhotoProps> = ({ header, imageUrl }) => {
+  return (
+    <Box sx={{ textAlign: 'center' }}>
+      <Typography
+        variant="h3"
+        sx={{
+          fontWeight: 700,
+          fontSize: '20px',
+          marginBottom: 6,
+        }}
+      >
+        {header}
+      </Typography>
```

Comment:
Could we use the `SettingHeader` component here instead, this will ensure we are matching from page to page.

Reply:
Updated to SettingHeader in latest version.
Also made slight updates to fontSize and margin within SettingsHeader.tsx

Latest screenshot below:
![Screen Shot 2022-01-17 at 10 08 25 PM](https://user-images.githubusercontent.com/96796802/149864225-c2d8bc26-24a6-41db-9405-8d4cf6b1897d.png)